src/GoogleAdWordsEventForwarder.js

```diff
@@ -166,6 +210,36 @@
             }
         }
 
+        function getUserConsentState() {
+            var userConsentState = null;
+
+            if (mParticle.Identity && mParticle.Identity.getCurrentUser) {
+                var currentUser = mParticle.Identity.getCurrentUser();
+
+                if (!currentUser) {
+                    return null;
+                }
+
+                var consentState =
+                    mParticle.Identity.getCurrentUser().getConsentState();
```

Comment:
can we actually be extra safe and do the check on window instead?

```suggestion
            if (window.mParticle && window.mParticle.Identity && window.mParticle.Identity.getCurrentUser) {
                var currentUser = window.mParticle.Identity.getCurrentUser();

                if (!currentUser) {
                    return null;
                }

                var consentState =
                    window.mParticle.Identity.getCurrentUser().getConsentState();
```

https://github.com/mparticle-integrations/mparticle-javascript-integration-leanplum/blob/master/src/LeanplumAnalyticsEventForwarder.js#L254-L268